packages/lib-classifier/src/plugins/tasks/DrawingTask/components/components/draggable/draggable.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react'
+
+function draggable (WrappedComponent) {
+  class Draggable extends Component {
+    constructor (props) {
+      super(props)
+      this.dragStart = this.dragStart.bind(this)
+      this.dragMove = this.dragMove.bind(this)
+      this.dragEnd = this.dragEnd.bind(this)
+      this.wrappedComponent = React.createRef()
+      this.state = {
+        coords: {
+          x: props.coords.x,
+          y: props.coords.y
+        },
+        dragging: false
+      }
+    }
+
+    convertEvent (event) {
+      const type = event.type
+
+      const svgEventOffset = this.getEventOffset(event.clientX, event.clientY)
+
+      const svgCoordinateEvent = {
+        pointerId: event.pointerId,
+        type,
+        x: svgEventOffset.x,
+        y: svgEventOffset.y
+      }
+
+      return svgCoordinateEvent
+    }
+
+    getEventOffset (x, y) {
+      const { svg } = this.props
+      const svgEvent = svg.createSVGPoint()
+      svgEvent.x = x
+      svgEvent.y = y
+      const svgEventOffset = svgEvent.matrixTransform(svg.getScreenCTM().inverse())
+      return svgEventOffset
+    }
+
+    dragStart (event) {
+      event.stopPropagation()
+      event.preventDefault()
+      const { setPointerCapture } = this.wrappedComponent.current
+      const { x, y, pointerId } = this.convertEvent(event)
+      this.setState({ coords: { x, y }, dragging: true })
+      this.props.dragStart({ x, y, pointerId })
+      setPointerCapture && this.wrappedComponent.current.setPointerCapture(pointerId)
+    }
+
+    dragMove (event) {
+      const { coords, dragging } = this.state
+      if (dragging) {
```

Comment:
Check the `pointerId` somewhere here and don't drag if it doesn't match the pointer that started the drag.